src/cli/cmds/synth.ts

```diff
@@ -75,4 +116,129 @@ async function fetchValidations(): Promise<ValidationConfig[] | undefined> {
 }
 
 
+async function helmSynthesis(apiVersion: string, chartVersion: string, outdir: string) {
+  const TEMPLATE_WITHOUT_CRDS = 'helm-chart-without-crds';
+  const TEMPLATE_WITH_CRDS = 'helm-chart-with-crds';
+
+  const pkgroot = path.join(__dirname, '..', '..', '..');
+  const templatesDir = path.join(pkgroot, 'templates');
+
+  let templatePath: string;
+
+  const substituteValues = {
+    apiVersion: apiVersion,
+    version: chartVersion,
+    app: path.basename(path.resolve()),
+  };
+
+  // Helm chart structure without CRDs support
+  if (apiVersion === HelmChartApiVersion.V1) {
+
+    if (config.imports && config.imports.length > 0) {
+      throw new Error(`CRDs are not supported in cdk8s for --chart-api-version: ${apiVersion}'. Please use ${HelmChartApiVersion.V2} for using CRDs.`);
+    }
+
+    templatePath = path.join(templatesDir, TEMPLATE_WITHOUT_CRDS);
+  } else {
+    templatePath = path.join(templatesDir, TEMPLATE_WITH_CRDS);
+  }
+
+  try {
+    await sscaff(templatePath, outdir, substituteValues);
+  } catch (error) {
+    throw new Error(`An error occurred during Helm chart creation: ${error}`);
+  }
+
+  await addCrdsToHelmChart(outdir);
+}
+
+async function addCrdsToHelmChart(chartDir: string) {
+  try {
+    const CRD_KIND = 'CustomResourceDefinition';
+    const manifestFiles = await downloadCrds();
+
+    for (const file of manifestFiles) {
+      const parsed = yaml.parseAllDocuments(file);
+
+      let filename;
+      for (const doc of parsed) {
+        const parsedToJson = doc.toJSON();
+        if (
+          parsedToJson.kind === CRD_KIND &&
+          parsedToJson.metadata &&
+          parsedToJson.metadata.name
+        ) {
+          filename = parsedToJson.metadata.name;
+        }
+      }
```

Comment:
Why do we need all these? Why not just put the downloaded file in the `crds` directory as is? and derive the filename from the URL? 

Or at least we should reuse the `ImportCustomResourceDefinition.fromSpec` function to perform the download and collect the CRDs.